rest/rest_index.go

```diff
@@ -558,7 +569,15 @@ func (h *CountPIndexHandler) ServeHTTP(
 			" no pindex, pindexName: %s", pindexName), http.StatusBadRequest)
 		return
 	}
-	if pindex.Dest == nil {
+	// check here for cold indexes
+	indexDef, _, err := h.mgr.GetIndexDef(pindex.IndexName, false)
```

Comment:
This need to be revisited? Don't think we need to guard against COLDness here?

Reply:
yep, my bad, I forgot to remove that comment.